### ISSUE TITLE
`and [Hibernation] and [Flash Food]` found Quest for Pure Flame, somehow

### ISSUE THREAD

**Issue by vorpal-buildbot** [PennyDreadfulMTG/Penny-Dreadful-Tools#4125]
Reported on Discord by bakert#2193

**Comment by jgabrielygalan** [comment]
Flash Food is not exact, it has a typo, it's  missing an 'l'. So it goes into the fuzzy matching and Flash is at distance 2 of Flame. The scores for each are: 
[('Flash Flood', 18.45685923226244), ('Quest for Pure Flame', 10.859564797095231)]

And writing this I realized that I've removed the normalized analyzer, so I broke other things, on top of creating the above effect. Things like 'Winds of Wrath' don't work anymore. I'm going to fix it.
